website/pages/blog/announcing-self-hosted-graphql-hive.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing self-hosted GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: [laurin]
+date: 2022-10-04
+description: Host your own GraphQL Hive instance.
+image: /blog-assets/announcing-self-hosted-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-self-hosted-graphql-hive/thumbnail.png
+---
+
+It has been 5 months since [we first announced GraphQL Hive](/blog/graphql-hive-preview), and it has been a wild ride since then!
+
+> GraphQL Hive is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics.
+> Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
+
+We first were surprised but also excited about the amount of interest from companies of all sizes and industries and individuals from all over the world in GraphQL Hive.
+We are so grateful for all the support, contributions and feedback we received so far from external contributors and are looking forward to a bright future!
+
+Aside from the diverse range of companies that are already successfully using our hosted GraphQL Hive version, a lot of companies have been asking us whether it is possible to host your own GraphQL Hive instance!
+
+Since the beginning of the project, we had a different vision for SaaS and community editions of GraphQL Hive compared to traditional SaaS products.
+Instead of only open-sourcing a small portion of the whole stack we always wanted to open-source the full stack and make it easy for anyone to host their own instance of GraphQL Hive.
+
+Today, we are happy to announce that we have reached this milestone and spinning up your first GraphQL Hive instance is as easy as running a single command!
+All you need is `docker` and `docker-compose` installed on your machine.
+
+[Head over to our GraphQL Hive documentation for getting started!](https://docs.graphql-hive.com/self-hosting/get-started)
```

Comment:
link is broken
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/7361780/193832576-a53a3153-0223-4bd4-90cb-6457bebdd0bd.png">

Reply:
@kamilkisiela Seems like the docs did not get redeployed 👀  https://docs.graphql-hive.com/